src/PyFOOOF.jl

```diff
@@ -0,0 +1,52 @@
+module PyFOOOF
+
+using PyCall
+
+#####
+##### Exports
+#####
+
+#####
+##### init
+#####
+
+const fooof = PyNULL()
+
+function __init__()
+    # all of this is __init__() so that it plays nice with precompilation
+    # see https://github.com/JuliaPy/PyCall.jl/#using-pycall-from-julia-modules
+    copy!(fooof, pyimport("fooof"))
+    # don't eval into the module while precompiling; this breaks precompilation
+    # of downstream modules (see #4)
```

Comment:
The issue reference here is for PyMNE, not this repo
```suggestion
    # of downstream modules
```